projects/plugins/jetpack/modules/videopress/class.videopress-player.php

```diff
@@ -671,7 +671,7 @@ function html5_dynamic_next() {
 
 			$js_url = 'https://s0.wp.com/wp-content/plugins/video/assets/js/next/videopress-iframe.js';
 
-			return "<iframe width='" . esc_attr( $videopress_options['width'] )
+			return "<iframe title='video player' aria-label='video player' width='" . esc_attr( $videopress_options['width'] )
```

Comment:
Shouldn't we localize these strings? And I had a look at YouTube, and they have `YouTube Video Player` so I'm thinking we could do `VideoPress Video Player`